src/Categories/Category/Dagger/Construction/DaggerFunctors.agda

```diff
@@ -0,0 +1,57 @@
+{-# OPTIONS --safe --without-K #-}
+
+module Categories.Category.Dagger.Construction.DaggerFunctors where
+
+open import Categories.Category.Dagger
+import Categories.Category.Construction.DaggerFunctors as cat
+open import Categories.Functor.Dagger
+open import Categories.NaturalTransformation
+open import Categories.NaturalTransformation.NaturalIsomorphism
+
+open import Level
+
+private
+  variable
+    o ℓ e o′ ℓ′ e′ : Level
+
+DaggerFunctors : (C : DaggerCategory o ℓ e) (D : DaggerCategory o′ ℓ′ e′)
+               → DaggerCategory (o ⊔ ℓ ⊔ e ⊔ o′ ⊔ ℓ′ ⊔ e′) (o ⊔ ℓ ⊔ ℓ′ ⊔ e′) (o ⊔ e′)
+DaggerFunctors C D = record
+  { C = cat.DaggerFunctors C D
+  ; hasDagger = record
```

Comment:
this would be better as part of the local `where` block, where you can also use a sub-local block (instead of `let`) and give names to the proofs instead of inlining them.

Reply:
I've moved the definition of `_†` to a where clause, which I think is the biggest benefit here. I think I'm now happy as this is; I don't think there's much benefit to giving names to the proofs for `commute` and `sym-commute` here.